lib/task.js

```diff
@@ -273,9 +275,43 @@ export class Task {
         .concat(oldEntrypoint);
     }
 
+    if(this.task.payload.features && this.task.payload.features.allowPtrace) {
```

Comment:
Ah, great, I'm glad a bug was already entered for that.  I haven't gotten to all my bugmail since PTO so thanks for pointing that one out.